### PR TITLE
Dash wrapper for SubsurfaceViewer

### PR DESCRIPTION
**Pull request by hkfb**
Allows mapping annotations to views in Dash.

Example:

```
annotation: {
    view_1: <ViewFooter>Hugin</ViewFooter>,
    view_2: <ViewFooter>kH Netmap</ViewFooter>,
}
```

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/SubsurfaceViewerDashWrapper.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        count: [291, 229],
+        increment: [25, 25],
+        rotDeg: 0,
+    },
+    propertiesUrl: "kh_netmap_25_m.float32",
+    contours: [0, 100],
+    material: false,
+};
+
+const Template: ComponentStory<typeof SubsurfaceViewerDashWrapper> = (args) => (
```
Comment (VincentNevermore) [inline react/src/lib/components/DeckGLMap/SubsurfaceViewerDashWrapper.stories.tsx:26]: Maybe we an call it DashWrapperTemplate to make it more precise than Template.

**Comment by VincentNevermore** [comment]
Maybe add a link for this PR to indicate which issue this PR will be solved.

File: react/src/lib/components/DeckGLMap/SubsurfaceViewerDashWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+        getTooltip: getTooltip,
+        cameraPosition: cameraPosition,
+        getCameraPosition: getCameraPosition,
+        triggerHome: triggerHome,
+        triggerResetMultipleWells: triggerResetMultipleWells,
+        children: children,
+    };
+
+    return (
+        <>
```
Comment (VincentNevermore) [inline react/src/lib/components/DeckGLMap/SubsurfaceViewerDashWrapper.tsx:133]: Think we do not need the <></> tag here as we do not need to wrap any JSX element.

**Comment by hkfb** [comment]
Btw, @HansKallekleiv I will create a new PR for generating the `ViewFooter` Dash component. Not sure what is the best way to do that yet. But this should work with other kinds of annotations, eg. `<div>Name</div>` or `<ColorLegend/>`

**Review by VincentNevermore — APPROVED** [review]

**Pull request merged**